pkg/execute/executor.go

```diff
@@ -325,20 +323,49 @@ func (e *DefaultExecutor) makeFiltersList() string {
 	return buf.String()
 }
 
+type kubectlVersionOutput struct {
+	Server struct {
+		GitVersion string `json:"gitVersion"`
+	} `json:"serverVersion"`
+}
+
+func (e *DefaultExecutor) findK8sVersion() (string, error) {
```

Comment:
Later we can think about extracting it to kubectl executor (as the version will be more related to a given executor). Not sure what about BotKube version though 🤔 Maybe separate command - `@BotKube ping / @BotKube version` will show just BotKube-specific data, and  `@BotKube kubectl version` K8s versions? Anyway, that's a topic for future discussion 🙂